src/databricks/labs/ucx/queries/assessment/interactive/01_0_compute_access_mode_limitation_summary.sql

```diff
@@ -0,0 +1,59 @@
+-- viz type=table, name=Compute Access Mode Limitation Summary, columns=issue,link,distinct_workspaces,distinct_notebooks,distinct_clusters,distinct_users
```

Comment:
this folder must not be in the `assessment` folder, as it'll break `assessment` job on no-uc workspaces.

Reply:
The `assessment` job doesn't call the `interactive` dashboard as of today. So we're safe for now. Will look at moving it as a safety precaution in a follow on PR.